Sources/YPersistence/SortInfo.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  SortInfo.swift
+//  YPersistence
+//
+//  Created by Sahil Saini on 17/04/23.
+//  Copyright © 2023 Y Media Labs. All rights reserved.
+//
+
+import Foundation
+
+/// Sort attribute (column) name info collection
+public struct SortInfo: Equatable {
+    /// Collection of the receivers, attribute (column) name
+    public let columns: [SortColumn]
+
+    /// Converts the collection of the receivers to a sort descriptor collection
```

Comment:
```suggestion
    /// Converts the receiver to an array of sort descriptors.
```
"receiver" means the current object. The instance.